### ISSUE TITLE
Responsive aka viewBox attribute

### ISSUE THREAD

**Issue by soukupl** [Yoctol/react-d3-cloud#95]
Hello,
would you please consider switching from width and height attributes of `<svg>` element to viewBox attribute (optionally / default) so the resulting `<svg>` is responsive?

So, instead of:
`<svg width="300" height="300">...</svg>`

it would be like this:
`<svg viewBox="0 0 300 300">...</svg>`

This would change the SVG canvas to be fully responsive and handled by browser. You will have your canvas set to specific "pixels" but it can be rendered at any width and height (with the same ratio) and allow browser to scale it for full responsive behavior.

**Comment by abbeyseto** [comment]
Yes, i quite agree. i have a challenge making the wordclout fit into the parent container because i cannnot set the viewbox attribute.
